mitmproxy/web/app.py

```diff
@@ -157,6 +159,26 @@ def get(self):
             data=[_strip_content(f.get_state()) for f in self.state.flows]
         ))
 
+class DumpFlows(RequestHandler):
+    def get(self):
+        self.set_header("Content-Description", "File Transfer")
+        self.set_header("Cache-Control", "no-cache, no-store, must-revalidate")
+        self.set_header("Content-Disposition", "attachment; filename=flows")
+        self.set_header("Content-Type", "application/octet-stream")
+        self.set_header("Content-Transfer-Encoding", "binary")
+
```

Comment:
Where does this list come from? :smile:
It feels like you can remove everything but `Content-Disposition` and `Content-Type`.